camille/output/bazefetcher.py

```diff
@@ -174,6 +143,10 @@ def __call__(self,
             to be written, is deleted.
             False - raise a ValueError on overwrite attempt.
             Default is False
+        fill : bool, optional
```

Comment:
How does it play with overwrite? 
ovewrite=false, fill = true - only insert new data
overwrite=true - totally overwrite the data, regardless of fill value